tests/integration/conftest.py

```diff
@@ -473,3 +475,85 @@ def _inner(source, **kwargs):
         workflow_id = infer_latest_run_from_id(workflow_id)
         return workflow_id
     yield _inner
+
+
+@pytest.fixture
+def complete():
```

Comment:
Unused.

Reply:
It was supposed to be used as per https://github.com/cylc/cylc-flow/pull/5923#discussion_r1453217855

Reply:
The test for completion was changed since then, the new test is reasonable providing it is constrained by a timeout. Otherwise switch to `complete()` which has a default timeout built in.